doc/ref_model/chapters/appendix-a.md

```diff
@@ -16,19 +16,75 @@ This Appendix has two goals:
 
-The goal is not to be prescriptive on how to re-architect existing or architect new applications but rather staying within scope of focusing on interface and interaction between applications and platform.
+The goal is not to be prescriptive on how to re-architect existing or architect new applications but rather staying within scope of focusing on interface and interaction between applications and platform. So, it will primarily look at Operational impact (in steps like Install, Configure, Validate, Operate) towards full lifecycle automation supporting CI/CD.
 
 <a name="A.2"></a>
 ## A.2 Intro and Terminology
-(Summary status and trends of ETSI NFV and Cloud. Decoupling application from platform. Terminology used/introduced.)
+(Summary status and trends of ETSI NFV and Cloud.)
+
+Terminology used or introduced:
+
+-	Decoupling, Loose Coupling = Loosely coupled system is one in which each of its components has, or makes use of, little or no knowledge of the definitions of other separate components. Loose coupling is the opposite of tight coupling. [1]
+
+-	Encapsulation = Restricting of direct access to some of an object's components. [2]
+
+-	Appliance deployment model = Application has tight coupling with underlying Platform even if the application is virtualized or containerized.
+
+-	Tenant/Cloud deployment model = Tenant brings applications that are decoupled from the platform provided by Cloud operator. Tenant and Cloud operator are normally different legal entities.
+
+-	Application Control = Any way of controlling tenant’s application. Depending on RA and technologies used, it can be tenant’s VNFM or Orchestration SW, or Platform capability.
+
+-	Decomposition = Decomposition (also known as factoring) is breaking a complex system into parts that are easier to program and maintain. [3]
+
+-	Resilience = Resilience is the ability to provide and maintain an acceptable level of service in the face of various faults and challenges to normal operation. [4]
+
+-	Observability = Observability is a measure of how well internal states of a system can be inferred from knowledge of its external outputs. [5]
+
 
 <a name="A.3"></a>
 ## A.3 VNF Maturity Levels
 (And how they map to CNTT RAs)
 
+Taking advantage of CNTT pre-defined environments with common capabilities, applications can be developed and deployed more rapidly, providing more service agility and easier operations. The extent to which this can be achieved will depend on levels of decoupling between application and infrastructure or platform underneath the application:
+
+**1. Infrastructure**:
+- a. Application functionality or application control require infrastructure components beyond RM profiles or requires infrastructure configuration changes beyond RA exposed APIs. Generally, such application is tightly coupled with infrastructure which results in Appliance deployment model. 
```

Comment:
Typo's suggest the following:
Application functionality or application control requires infrastructure components beyond RM profiles or infrastructure configuration changes beyond RA exposed APIs. Generally, such an application is tightly coupled with the infrastructure which results in an Appliance deployment model.

Reply:
Done in #1376.